Brickwall/models.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy import Column, Integer, String, DateTime, Boolean, ForeignKey
+from . import db
+from enum import Enum
+
+
+
+class Review(db.Model):
+    __tablename__ = "review"
+    review_id = Column(Integer, primary_key=True)
+    # user name of reviewer
+    member = Column(String)
```

Comment:
All strings should be bound in some way by a size
Example: https://github.com/ComputerScienceHouse/packet/blob/develop/packet/models.py#L133

Reply:
How do I know what the size should be/ what is a good rule of thumb for choosing string size?

Reply:
For things like uid's, a lot of services define it as a max of 55. At the very least cap most strings at 256 unless we want it to be a large (like the body)

Reply:
ok, what should the body be? Would a few thousand characters be enough?